source/Body.h

```diff
@@ -88,13 +89,17 @@ class Body {
 	void MarkForRemoval();
 	// Mark that this object should not be removed (e.g. a launched fighter).
 	void UnmarkForRemoval();
+	// Turn this object its around center of rotation.
+	void Turn(double amount);
```

Comment:
Thoughts on having two turn functions for ease of use, instead of needing the caller to convert angles to degrees?
```suggestion
	void Turn(double amount);
	void Turn(const Angle &amount);
```

Reply:
ye sure